lib/blob.js

```diff
@@ -744,25 +859,31 @@ Blob.prototype.getContainerProperties = function getContainerProperties(name, le
  * }
  * ```
  */
-Blob.prototype.getContainerACL = function getContainerACL(name, leaseId){
-  assert(name, 'The name of the container must be specified');
+Blob.prototype.getContainerACL = function getContainerACL(name, options){
+  assert(typeof name === 'string', 'The name of the container must be specified and must be a string value.');
   var query = {
     restype: 'container',
     comp: 'acl'
   }
   var path = '/' + name;
 
   var headers = {};
-  if (leaseId) {
-    assert(utils.isValidGUID(leaseId), '`leaseId` is not a valid GUID.');
-    headers['x-ms-lease-id'] = leaseId;
+  if (options && options.leaseId) {
+    assert(utils.isValidGUID(options.leaseId), '`leaseId` is not a valid GUID.');
+    headers['x-ms-lease-id'] = options.leaseId;
   }
 
   return this.request('GET', path, query, headers).then(function (response) {
     if (response.statusCode !== 200) {
       throw new Error("getContainerACL: Unexpected statusCode: " + response.statusCode);
     }
-    return xml.blobParseContainerACL(response);
+    var result = xml.blobParseContainerACL(response);
```

Comment:
isn't this the same? or maybe I'm missing something...
Note: I also prefer `return {...}` over `var result = {}; result... = ...; return result;`